pilot/pkg/model/service.go

```diff
@@ -239,7 +239,7 @@ type WorkloadInstance struct {
 	Name      string            `json:"name,omitempty"`
 	Namespace string            `json:"namespace,omitempty"`
 	Endpoint  *IstioEndpoint    `json:"endpoint,omitempty"`
-	PortMap   map[string]uint32 `json:"portMap,omitempty"`
+	PortMap   map[string]uint32 `json:"GatewayPortMap,omitempty"`
```

Comment:
Are we using the external json representation anywhere ? Debug, istioctl, etc ? If yes - not backward compatible.

Reply:
Refactoring gone wrong, will revert

Reply:
although I don't think we do expose it fwiw